schema_utils_test.py

```diff
@@ -433,6 +433,49 @@ def test_schemas_are_correctly_validated(self):
             with self.assertRaises((AssertionError, KeyError)):
                 validate_schema(schema)
 
+    def test_normalize_against_schema_raises_exception(self):
+        """Tests if normalize against schema raises exception
+        for invalid key.
+        """
+        with self.assertRaises(Exception):
+            schema = {SCHEMA_KEY_TYPE: 'invalid'}
+            schema_utils.normalize_against_schema('obj', schema)
+
+    def test_is_nonempty_validator(self):
+        """Tests if static method is_nonempty returns true iff obj
+        is not empty str.
+        """
+        is_nonempty = schema_utils.get_validator('is_nonempty')
+        self.assertTrue(is_nonempty('non-empty string'))
+        self.assertTrue(is_nonempty(' '))
+        self.assertTrue(is_nonempty('    '))
+        self.assertFalse(is_nonempty(''))
+
+    def test_is_at_most_validator(self):
+        """Tests if static method is_at_most returns true iff obj
+        is at most a value.
+        """
+        is_at_most = schema_utils.get_validator('is_at_most')
+        self.assertTrue(is_at_most(2, 3))
+        self.assertTrue(is_at_most(2, 2)) # boundary
+        self.assertFalse(is_at_most(2, 1))
+
+    def test_has_length_at_least_validator(self):
+        """Tests if static method has_length_at_least returns true iff
+        given list has length of at least the given value.
+        """
+        has_len_at_least = schema_utils.get_validator('has_length_at_least')
+        self.assertTrue(has_len_at_least(['elem'], 0))
+        self.assertTrue(has_len_at_least(['elem'], 1)) # boundary
+        self.assertFalse(has_len_at_least(['elem'], 2))
+
+    def test_get_raises_invalid_validator_id(self):
+        """Test if class method get in _Validator raises exception
```

Comment:
Test --> Tests (for consistency :) ) and get --> 'get'

Reply:
Addressed in most recent commit!